1.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <script>
+        //Task 1
+        function loop(times = 0, callback = null) {
+            if (callback) {
+                for (let i = 0; i < times; i++) {
+                    callback();
+                }
+            }
+        }
+        //Task 2
+        function calculateArea() {
+            let figure = '';
+            let area = 0;
+            let input = [...arguments];
+            switch (arguments.length) {
+                case 3: {
+                    figure = 'Треугольник';
+                    //тут будут вычисления высоты и площади треугольника
+                    break;
+                };
+                case 4: {
+                    console.log('Квадрат');
+                    break;
+                }
+                default: {
+                    console.log('Не верный ввод');
+                    break;
+                }
+            }
+            return [area, figure, input];
```

Comment:
```suggestion
            return {area, figure, input};
```
надо возвращать объект